### PR TITLE
UI: Add missing tooltips on service offering creation

### PR DESCRIPTION
**Pull request by nvazquez**
### Description

This PR adds the missing tooltips helpers to fields in the service offering creation: 'Compute only disk offering' and 'Disk offering stricteness'

Screen Shot 2022-05-11 at 09 12 04

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

File: ui/src/views/offering/AddComputeOffering.vue [hunk 1]
```diff
@@ -340,7 +340,7 @@
         </a-form-item>
         <a-form-item name="computeonly" ref="computeonly">
           <template #label>
-            {{ $t('label.computeonly.offering') }}
+            <tooltip-label :title="$t('label.computeonly.offering')" tooltip="Option to specify root disk related information in the compute offering or to link a disk offering to the compute offering"/>
```
Comment (sureshanaparti) [inline ui/src/views/offering/AddComputeOffering.vue:343]:
```suggestion
            <tooltip-label :title="$t('label.computeonly.offering')" tooltip="Option to directly map root disk related information to the compute offering"/>
```

cc @harikrishna-patnala please advise

Comment (Pearl1594) [inline ui/src/views/offering/AddComputeOffering.vue:343]: @nvazquez can you add the tooltip message to en.json file?

**Comment by andrijapanicsb** [comment]
What about "Disk offering strictness" checkbox?

Any reason to say that it won't work with deploy-as-is templates unless we are using custom disk size, etc, @nvazquez ?

File: ui/src/views/offering/AddComputeOffering.vue (hunk 1 above)
Comment (sureshanaparti) [reply ui/src/views/offering/AddComputeOffering.vue]: I think, most of the tooltip descriptions are from the API params, and are not translated. In that case, this is in consistent with the other tooltips.

Comment (nvazquez) [reply ui/src/views/offering/AddComputeOffering.vue]: Sorry @sureshanaparti have missed your comment - it is already fixed

**Comment by andrijapanicsb** [comment]
Looks good on the QA server (disk strictness checkbox is missing, since it's 4.16.1 based/ACS)

I would like to have this disk strictness also with tooltip - do we have links when user click on that, that leads to the documentation (as with other tooltips?)

This one here: 
image

I see (if I see correctly...) that the Decoupling of disk from service offering documentation is missing?
http://docs.cloudstack.apache.org/en/latest/adminguide/service_offerings.html#creating-a-new-compute-offering
the main branch also is missing it: https://github.com/apache/cloudstack-documentation/blob/main/source/adminguide/service_offerings.rst 

This needs be corrected (asynchronously from this PR) @harikrishna-patnala cc @nvazquez

**Comment by nvazquez** [comment]
@andrijapanicsb the tooltip has been added to the disk strictness. There is a documentation PR for that change which is still in progress cc @harikrishna-patnala https://github.com/apache/cloudstack-documentation/pull/238

File: ui/src/views/offering/AddComputeOffering.vue (hunk 1 above)
Comment (harikrishna-patnala) [reply ui/src/views/offering/AddComputeOffering.vue]: @sureshanaparti, I think it is better to mention both cases.

how about "Option to map root disk related information directly with a disk offering". If we say this we might have to change the toggle button from "Compute only offering" to "Map to a disk offering" and keep it off by default.

**Comment by harikrishna-patnala** [comment]
> 

It did not say anything about deploy as is templates

> Looks good on the QA server (disk strictness checkbox is missing, since it's 4.16.1 based/ACS)
> 
> I would like to have this disk strictness also with tooltip - do we have links when user click on that, that leads to the documentation (as with other tooltips?)
> 
> This one here:
> 
> image
> 
> I see (if I see correctly...) that the Decoupling of disk from service offering documentation is missing? http://docs.cloudstack.apache.org/en/latest/adminguide/service_offerings.html#creating-a-new-compute-offering the main branch also is missing it: https://github.com/apache/cloudstack-documentation/blob/main/source/adminguide/service_offerings.rst
> 
> This needs be corrected (asynchronously from this PR) @harikrishna-patnala cc @nvazquez

I'm working on the doc PR, will update it asap.

Regarding "disk offering strictness" missing, I see it on other environments along with this PR changes also. So may be some issue with qa server not up to date.

**Review by shwstppr — APPROVED** [review]
Minor comment on translation key otherwise change works fine
Screenshot from 2022-05-12 11-42-07

File: ui/src/views/offering/AddComputeOffering.vue (hunk 1 above)
Comment (sureshanaparti) [reply ui/src/views/offering/AddComputeOffering.vue]: ok @harikrishna-patnala if both cases makes sense for "Compute only offering", keep that.

**Review by harikrishna-patnala — APPROVED** [review]
LGTM

**Review by sureshanaparti — APPROVED** [review]

**Comment by andrijapanicsb** [comment]
@blueorangutan build

**Comment by andrijapanicsb** [comment]
@blueorangutan help

**Comment by andrijapanicsb** [comment]
@blueorangutan package

**Comment by andrijapanicsb** [comment]
Come on @blueorangutan , you can do it, it's been a while....

**Review by andrijapanicsb — APPROVED** [review]
LGTM

**Pull request merged**